src/GeocodeParser.js

```diff
@@ -15,9 +16,24 @@ function __internals_get_results(data = {}) {
   return results ? results[0] : null;
 }
 
+function parseRoute(route) {
+  if (route) {
+    return {
+      preDirectional: checkPredirectional(route),
```

Comment:
Could we remove the suffix? `predirection`, `postdirection`, `checkPredirection`,`checkPostdirection`, `getPredirection`, `getPostdirection`